.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,14 @@
 <!--
+IMPORTANT NOTICE FOR NEW CONTRIBUTORS: This is the repository of the Python library 
```

Comment:
This is the issue template. Not for contributers right?

Reply:
Wasn't sure what to call it :/ You got an idea?